husky_components/column-navigation/main.js

```diff
@@ -1587,6 +1640,18 @@ define(function() {
             } else {
                 this.sandbox.emit(ACTION.call(this), item);
             }
+        },
+
+        /**
+         * Returns all the marked objects.
+         */
+        getMarkedHandler: function() {
+            var elements = [];
+            $.each(this.marked, function (key, markedElement) {
```

Comment:
I would say the [map function](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/map) is a better match here.